README.md

```diff
@@ -158,3 +158,16 @@ In this case, one solution is:
 ```
 conda install protobuf
 ```
+
+## Citation
+
+Please cite:
+
+```bibtex
+@inproceedings{galactic,
+  author={Vincent-Pierre Berges and Andrew Szot and Devendra Singh Chaplot and Aaron Gokaslan and Roozbeh Mottaghi and Dhruv Batra and Eric Undersander},
```

Comment:
Is there someway to indicate equal contribution for you and Andrew here, as we did in the paper?